wasm/service/kuasar-wasm.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Kuasar Wasm sandboxer daemon process
+After=network.target
+
+[Service]
+Type=simple
+ExecStart=/usr/local/bin/wasm-sandboxer --listen /run/wasm-sandboxer.sock --dir /run/kuasar-wasm
+ExecReload=/bin/kill -s HUP $MAINPID
```

Comment:
Currently wasm type sandboxer doesn't support config reload feature, `ExecReload` needs to be removed.

Reply:
What is the consequence of keeping this `ExecReload`?

Reply:
If the user runs the `systemctl reload wasm-sandboxer` command, the wasm-sandboxer process will receive the SIGHUP signal and process exit abnornally, which does not meet the expectation of the `systemctl reload` command.

Reply:
fixed